### ISSUE TITLE
508-defect-3 [SCREENREADER]: The same terms SHOULD be used for non-sighted and sighted users

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12547]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

There is an `aria-label="Name of institution"` on the text input that pairs with the label that reads "School name". These two **should** be aligned, along with anywhere else in the interface that uses 'institution' for screen reader users but 'school' for sighted users.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user with low vision, I want to have consistency in experience so that I am not confused as I use the form.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: production

## Steps to Recreate

1. Enter https://www.va.gov/education/yellow-ribbon-participating-schools/  in browser
2. Start screenreading device listed in Environment
3. Navigate to form
4. Verify that the label reads School name and the screen reader says Name of institution

## Possible Fixes (optional)

Recommend removing the `aria-label` attributes from the form inputs, because the label's `for` attribute is paired with the input's `id` value, so the aria-label is redundant. There are similar aria-labels on form elements, such as the state search select element, and City input.

## WCAG or Vendor Guidance (optional)

* [WUHCAG 3.2.4 – Consistent Identification (Level AA)](https://www.wuhcag.com/consistent-identification/)

## Screenshots or Trace Logs

![Screen Shot 2020-08-04 at 12 23 54 PM (2)](https://user-images.githubusercontent.com/57469/90428331-d2d0cb00-e091-11ea-974b-cb0027a55eab.png)

![Screen Shot 2020-08-17 at 2 00 07 PM](https://user-images.githubusercontent.com/57469/90428580-32c77180-e092-11ea-905f-b39e26eb2a07.png)

**Comment by 1Copenut** [comment]
I concur, removing the `aria-labels` and using the paired `label[for="input-1"]` and `input[id="input-1"]` is the right approach. That keeps the visual labels and aural announcements 1:1.

**Comment by MarciMcGuireGCIO** [comment]
Moving to a consolidated 508 epic for Yellow Ribbon to clean up our backlog.

**Comment by zacharymorel** [comment]
@joshkimux this is now live on staging.

**Comment by joshkimux** [comment]
<img width="1508" alt="Screen Shot 2021-06-29 at 2 36 42 PM" src="https://user-images.githubusercontent.com/14154792/123849970-6ea0a980-d8e7-11eb-9e3b-cbbcd5c0632b.png">

Closing out 🥳

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @zacharymorel and @joshkimux ! 🥳
